docs/wiki/99_Developers/Developers_Guide.md

```diff
@@ -0,0 +1,219 @@
+# LORIS Developers Guide
+This document covers topics including making PRs, issues, and how to go through a normal LORIS workflow when contributing to the LORIS repository. Many notes in this guide are also relevant to contributing code on a fork or instance of LORIS, and many practices described here will be familiar to experienced developers.
+
+## Key Links
+Important standards and practices documented in our repo include:
+* [Contributing.md](../../../CONTRIBUTING.md) - How to contribute to Loris
+* [Coding Standards](../../CodingStandards.md)
+* [Pull Request Checklist](https://github.com/aces/Loris/wiki/Code-Review-Checklist) and [Code Review guidelines](Code_Review_Guide.md)
+
+## 1. Working off your fork
+During installation, you made your own fork of LORIS. Now you can add two remote options in order to switch between your fork and “aces”.
+
+    git remote rename origin aces
+    git remote add my_fork <fork_url>
+             
+Your fork url will usually have the format:  
+    `https://github.com/github-username/LORIS`  
+    
+Any development should be done on your fork of LORIS, and **not** on aces/LORIS. 
+Before starting any PR, be mindful of which branch you want to base off of (main vs release-specific branch).   
+
+You will make a different branch for each PR or change that you are making.  
+
+In general, smaller pull requests or changes are easier to work with in our team review context and will flow more smoothly through the review process.  
+
+Here is a basic workflow if you are trying to make a PR off of the _main_ branch:
+
+    git fetch aces
+    git checkout aces/main 
+    git checkout -b "branch_name"
+    vi filename
+    git add filename 
+    git status 
+    git commit -m "Initial commit"
+    git push my_fork branch_name
+
+There is usually no need to fully update your fork to the newest aces/LORIS version, since you can work off whichever branch you just fetched and checked-out.
+
+ **Never** push to aces/Loris. Your branch will be deleted. Always push to your fork, and then create the PR from GitHub.
+
+#### Branch Naming
+Follow the convention most LORIS developers use: “date-title...”.   
```

Comment:
I don't think that's a convention that most LORIS developers use

Reply:
it's a convention we appreciate and ask new developers to follow, part of Rida's conventional wisdom. it's also useful for new members.